### DISCUSSION TITLE
control the point of collision

### DISCUSSION THREAD

**Discussion by shivanimall** [bulletphysics/bullet3#3804] [General]
Currently, the way collision can be simulated seems somewhat out of user control. Here is what I mean: I can make an object (say B) fall from a certain base orientation & position in WCS; this results in contact forces at an arbitrary point of collision.

However, I am trying to approach this in a reverse manner, such that I want to create collision at a specific point on object B instead of letting collision happen at any arbitrary point on the object. Is there a direct way to solve this? 

Please let me know if my question is unclear, thank you!

**Comment by erwincoumans** [comment]
Manually adding a contact point? It is not supported at the moment, it could be done by modifying the  c++ code.
